source/features/add-updated-date-to-pinned-issues.tsx

```diff
@@ -0,0 +1,69 @@
+import clockIcon from 'octicon/clock.svg';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import select from 'select-dom';
+
+async function init(): Promise<false | void | any> {
+	// Get issues links that don't already have a specific sorting applied
+	for (const link of select.all<HTMLAnchorElement | any>('.pinned-issue-item')) {
```

Comment:
This is not a link, it's a plain div
```suggestion
	for (const pinnedIssue of select.all('.pinned-issue-item')) {
```